CMakeLists.txt

```diff
@@ -7,18 +7,22 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_BUILD_TYPE Release)
 
 # Avoid building tooling we won't need for release
-# See all options with `cmake -LA` in an `h3/build` directory
-set(BUILD_ALLOC_TESTS OFF CACHE BOOL "" FORCE)
-set(BUILD_BENCHMARKS OFF CACHE BOOL "" FORCE)
-set(BUILD_FILTERS OFF CACHE BOOL "" FORCE)
-set(BUILD_FUZZERS OFF CACHE BOOL "" FORCE)
-set(BUILD_GENERATORS OFF CACHE BOOL "" FORCE)
-set(BUILD_TESTING OFF CACHE BOOL "" FORCE)
-set(ENABLE_COVERAGE OFF CACHE BOOL "" FORCE)
-set(ENABLE_DOCS OFF CACHE BOOL "" FORCE)
-set(ENABLE_FORMAT OFF CACHE BOOL "" FORCE)
-set(ENABLE_LIBFUZZER OFF CACHE BOOL "" FORCE)
-set(ENABLE_LINTING OFF CACHE BOOL "" FORCE)
+# See all options with `cmake -LA` in an `h3/build` directory,
+# or at https://h3geo.org/docs/next/core-library/compilation-options/
+macro(turn_off option_name)
+    set(${option_name} OFF CACHE BOOL "" FORCE)
+endmacro()
+turn_off(BUILD_ALLOC_TESTS)
+turn_off(BUILD_BENCHMARKS)
+turn_off(BUILD_FILTERS)
+turn_off(BUILD_FUZZERS)
+turn_off(BUILD_GENERATORS)
+turn_off(BUILD_TESTING)
+turn_off(ENABLE_COVERAGE)
+turn_off(ENABLE_DOCS)
+turn_off(ENABLE_FORMAT)
+turn_off(ENABLE_LIBFUZZER)
+turn_off(ENABLE_LINTING)
```

Comment:
Just curious, can you compare compilation times before and after this change? Does this make it a lot faster?

Reply:
I spot checked some other compilation times. There's *maybe* a gain. I'll try some runs here for a clearer comparison.

Reply:
Seems to be faster, based on the last two runs I looked at, but hard to tell how much is random variation.

**Old config**
<img width="332" alt="Screen Shot 2022-06-06 at 3 51 11 PM" src="https://user-images.githubusercontent.com/984039/172262214-a39cf179-e9ee-4103-9838-92cb838d2eb8.png">

**New config**
<img width="320" alt="Screen Shot 2022-06-06 at 3 51 40 PM" src="https://user-images.githubusercontent.com/984039/172262215-f084324a-48dc-46a3-9283-adfdcd9cea91.png">